lib/travis/build/script/node_js.rb

```diff
@@ -2,10 +2,6 @@ module Travis
   module Build
     class Script
       class NodeJs < Script
-        DEFAULTS = {
```

Comment:
Bringing this back means there is always a ```config[:node_js]``` property(?) so if the user didn't specify a version it will be set to:
```
node_js:
  - 0.10
```

So we will never end up searching for a ```.nvmrc``` file